### PR TITLE
Fix robot api test add sleep

### PR DESCRIPTION
**Pull request by Thykof**

### CODE REVIEW

File: .github/workflows/api.yml [hunk 1]
```diff
@@ -54,6 +52,10 @@ jobs:
         run: xvfb-run ./main &
       - name: Define my.massa
         run: sudo echo "127.0.0.1 my.massa" | sudo tee -a /etc/hosts
+      - name: Wait for server to be up
+        uses: juliangruber/sleep-action@v1
+        with:
+          time: 25s
```
Comment (thomas-senechal) [inline .github/workflows/api.yml:58]: Isn't 25 seconds a bit overkill ? 😅 10 seconds should be enough, don't you think so ?

It's not blocking but we will lose a lot of time so I think we should make the sleep as short as possible and then make it longer if needed

Comment (peterjah) [reply .github/workflows/api.yml:58]: maybe we can ping my.massa and wait until it responds

Comment (Thykof) [reply .github/workflows/api.yml:58]: I tried to ping thyra on http://localhost, http://my.massa/thyra/home without success

Comment (Thykof) [reply .github/workflows/api.yml:58]: 15sec diff is nothing compared to uses:install/action.yml and other stuff

Comment (thomas-senechal) [reply .github/workflows/api.yml:58]: Let's go with this then. 
At least for now to unlock other issues.

**Review by thomas-senechal — APPROVED** [review]

File: .github/workflows/api.yml (hunk 1 above)
Comment (Thykof) [reply .github/workflows/api.yml:58]: @peterjah see my attempt here https://github.com/massalabs/thyra/pull/490/commits/1ca36d98f5a3beab33afb592a502e8c633e36754

**Review by peterjah — APPROVED** [review]

**Pull request merged**